frontend/ui/widget/buttondialog.lua

```diff
@@ -122,7 +122,7 @@ function ButtonDialog:init()
         shrink_min_width = self.shrink_min_width,
         show_parent = self,
     }
-    local buttontable_width = self.buttontable:getSize().w -- may be shrinked
+    local buttontable_width = self.buttontable:getSize().w -- may be shrunk
```

Comment:
May have shrunk?

Reply:
No, it means the `buttontable_width` we are computing here, may later get to be smaller.